scanner/src/main/kotlin/Main.kt

```diff
@@ -255,11 +256,15 @@ object Main {
 
             val results = scanner.scan(packages, outputDir, downloadDir)
             results.forEach { pkg, result ->
+                // TODO: Make output format configurable.
+                val resultsFile = File(outputDir, "scanResults/${pkg.id.toPath()}/scan-results.yml")
+                yamlMapper.writeValue(resultsFile, ScanResults(pkg.id, result))
```

Comment:
Also use `.also {}` here?

Reply:
Done, saves the variable assignment in this case.